libs/watt/package/modal/watt-modal.component.html

```diff
@@ -26,7 +26,12 @@
       />
     }
     @if (title && loading === false) {
-      <h2 class="watt-modal-title">{{ title }}</h2>
+      <h2 class="watt-modal-title">
+        @if (titleIcon) {
+          <watt-icon [name]="titleIcon" [size]="'l'" />
+        }
+        {{ title }}
+      </h2>
```

Comment:
issue: the title's styling needs to be adjusted as well because the icon has `display: flex` which causes the text to break on a new line.

![image](https://github.com/user-attachments/assets/b9387860-d71b-4d24-a2f3-d28c84d9883d)